SU-HSDO/suhumsci
# Created config split for not live modules such as nobots

# READY FOR REVIEW

# Summary
- Created a config split to hold any config/module settings for sites on production that aren't live.

# Needed By (Date)
- Sept 27

# Urgency
- High

# Steps to Test
1. checkout this branch
1. edit the file `vendor/acquia/blt/settings/config.settings.php` on line 87 add `$split = 'stage';`
1. sync archaeology `blt drupal:sync --site=archaeology`
1. ensure nobots is turned on `drush @archaeology.local pml | grep nobots`
1. change line 87 from earlier to `$split = 'prod';`
1. config import `drush @archaeology.local cim -y`
1. validate nobots is still enabled `drush @archaeology.local pml | grep nobots`
1. disable the config split `drush @archaeology.local cset config_split.config_split.not_live status 0`
1. config import `drush @archaeology.local cim -y`
1. validate nobots is disabled. `drush @archaeology.local pml | grep nobots`


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
